### ISSUE TITLE
Why isn't this in the Obsidian Plugins List

### ISSUE THREAD

**Issue by hwittenborn** [stravo1/obsidian-gdrive-sync#4]
Hey! I'm wanting to use this to sync my notes, but I'm wanting to avoid manually installing it as I'd like to receive updates from Obsidian itself automatically.

I don't know a ton about Obsidian yet, but it's of my knowledge that adding this to the Plugins List would fix such. Is there any reason that hasn't been able to happen yet?

**Comment by stravo1** [comment]
it is very much in beta and not ready for "public" release, and will require a lot of testing and changes before it can be listed on the official plugins' list. there's a genuine risk of data loss (although i have tried my best to cover all edge cases but can't risk other people's data solely on my coding skills 😅).

tldr: personally i don't want an unfinished plug-in with risk of data loss being listed as "official" :)

**Comment by kallekn** [comment]
Thanks for working with this, very much needed! But how do I install it on my Android device?

**Comment by stravo1** [comment]
> Thanks for working with this, very much needed! But how do I install it on my Android device?

https://github.com/stravo1/obsidian-gdrive-sync/assets/76593476/bfcb07ed-1764-4675-ba4e-3247e6313d7c

**Comment by stravo1** [comment]
@kallekn the steps are similar to how you would do it on a desktop, downloading the zip, extracting it, copying it to your vault location and storing it under ".obsidian/plugins"

![Screenshot_20240327-131555.png](https://github.com/stravo1/obsidian-gdrive-sync/assets/76593476/5905d8b1-3d0b-46e4-9f6b-7a492711eb65)

![Screenshot_20240327-132035.png](https://github.com/stravo1/obsidian-gdrive-sync/assets/76593476/9f2817d3-58b1-4a1b-9f7c-d71531d261ef)

**Comment by avichou** [comment]
> Android

do the vaults need to have the same name or does it not matter ?

**Comment by Audiophile-Official** [comment]
@stravo1 do you think obsidian will take issue with bypassing their own sync, which is the only way they make money? Regardless, super appreciative of your work!

**Comment by stravo1** [comment]
there are alternative syncing options on the plug-in list including Git, Dropbox, OneDrive etc., but very few are such hassle-free and plug-n-play as this one (although a lot of bugs to iron out and features to add), so it's a 50-50 chance maybe 🤔

**Comment by avichou** [comment]
> @stravo1 do you think obsidian will take issue with bypassing their own sync, which is the only way they make money? Regardless, super appreciative of your work!

curious, how do you see them preventing it , in case it does bother them  ? by breaking internal file management or excluding google drive ? or literally by asking stravo to remove his plugin ? i dont think the latter is allowed is it

**Comment by peanutprime** [comment]
![image](https://github.com/user-attachments/assets/9647530d-7abe-4236-a9fc-45889e0c50e2)
I setup Google Drive Sync for both my PC and my Android. I'm having issues Initializing Vault from my PC. When I click Proceed, it seems to "upload" files and once completed I see a new "Obsidian Vault (1)" folder. My original vault name is "Obsidian Vault." After, I went back to the Community Plugins to reload, then checked the Google Drive Sync plugin setting and seems like the vault wasn't initialized because I still see the "Proceed" button available. 
![image](https://github.com/user-attachments/assets/57f30149-e9a7-4d48-b307-464c813769cf)
Can you kindly advise what I need to do to sync it properly?

**Comment by stravo1** [comment]
i would suggest deleting ALL "Obsidian Vault" folders from Drive (even from Trash/Bin) and trying again. seems like there is some kind of folder sync enabled between your PC and Drive, might be an issue with that too.

if possible, please join the discord server for further discussions.

**Comment by peanutprime** [comment]
> i would suggest deleting ALL "Obsidian Vault" folders from Drive (even from Trash/Bin) and trying again. seems like there is some kind of folder sync enabled between your PC and Drive, might be an issue with that too.
> 
> if possible, please join the discord server for further discussions.

Hi @stravo1 I was able to figure it out. On Google Drive for Desktop, there is an option under Preferences to either Stream Files or Mirror Files. It has to be on Stream Files, I believe. Also I had to turn off Sync Obsidian option. Thought I'd share in case someone else ran into the same issues.
